src/components/Checkbox_Slider/CheckBoxSlide.jsx

```diff
@@ -0,0 +1,84 @@
+import {useState} from 'react';
+import styled, {css} from 'styled-components';
+
+function CheckBoxSlide({value, setValue}) {
+	const [isChecked, setIsChecked] = useState(value);
+	const handleCheck = e => {
```

Comment:
It is better to choose event instead of e.

Reply:
Done